src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mpishi</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <a href="#">
+            <img src="./Asset/restaurantLogoB.png" alt="MPISHI">
+        </a>
+        <nav class="nav-div">
+            <a class="menu" href="">Menu</a>
+        </nav>
+    </header>
+    <main>
+              <section class="menu-section">
+            <div class="menu-container">                         
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer">
+            <small>
+              Copyright &copy;
+              <script>
+                document.write(new Date().getFullYear());
+              </script>
+              Built by Fatuma Hussein & Mustakim Masum<br/>
+              All Rights Reserved
+            </small>
```

Comment:
- [ ] Kindly align the footer to the bottom of the page and not overlap. I have attached an image for your reference.

![footer](https://github.com/firepicaso/Social-Interaction-Capstone-Module2/assets/53124548/10d9f9d8-e5fd-492a-885d-06377778773d)


![footer1](https://github.com/firepicaso/Social-Interaction-Capstone-Module2/assets/53124548/ccbb1461-d1f6-4376-adb7-240ececf7063)